frontend/packages/dev-console/integration-tests/features/Knative/create-event-sources.feature

```diff
@@ -11,7 +11,12 @@ Feature: Create event sources
    Scenario: Different event source enters display in event sources add page : Kn-07-TC03, Kn-08-TC02
       Given user is at Add page
       When user clicks on "Event Source" card
-      Then user will be redirected to page with header name "Event Sources"
+      Then user will get redirected to Developer Catalog page
+      And user will see header name "Event Sources"
+      And user will see the list of Providers
+      And user will see the cards of Event Sources
+      And user will see Filter by Keyword field
+      And user will A-Z, Z-A sort by dropdown
```

Comment:
This is confusing. Is this what you mean here:
```suggestion
      And user will see A-Z, Z-A sort by dropdown
```

Reply:
Updated